src/Linux/Avalonia.LinuxFramebuffer/LinuxFramebufferPlatform.cs

```diff
@@ -13,49 +14,61 @@
 using Avalonia.OpenGL;
 using Avalonia.Platform;
 using Avalonia.Rendering;
-using Avalonia.Threading;
 
 namespace Avalonia.LinuxFramebuffer
 {
-    class LinuxFramebufferPlatform
+    internal class LinuxFramebufferPlatform
     {
-        IOutputBackend _fb;
+        private static readonly KeyboardDevice KeyboardDevice = new KeyboardDevice();
+        private static readonly MouseDevice MouseDevice = new MouseDevice();
+        private static readonly InternalPlatformThreadingInterface Threading = new InternalPlatformThreadingInterface();
+
         private static readonly Stopwatch St = Stopwatch.StartNew();
+        
         internal static uint Timestamp => (uint)St.ElapsedTicks;
-        public static InternalPlatformThreadingInterface Threading;
-        LinuxFramebufferPlatform(IOutputBackend backend)
+
+        private LinuxFramebufferPlatform()
         {
-            _fb = backend;
         }
 
-
-        void Initialize()
+        private void Initialize()
         {
-            Threading = new InternalPlatformThreadingInterface();
-            if (_fb is IGlOutputBackend gl)
-                AvaloniaLocator.CurrentMutable.Bind<IPlatformOpenGlInterface>().ToConstant(gl.PlatformOpenGlInterface);
             AvaloniaLocator.CurrentMutable
                 .Bind<IPlatformThreadingInterface>().ToConstant(Threading)
                 .Bind<IRenderTimer>().ToConstant(new DefaultRenderTimer(60))
                 .Bind<IRenderLoop>().ToConstant(new RenderLoop())
                 .Bind<ICursorFactory>().ToTransient<CursorFactoryStub>()
-                .Bind<IKeyboardDevice>().ToConstant(new KeyboardDevice())
+                .Bind<IKeyboardDevice>().ToConstant(KeyboardDevice)
+                .Bind<IMouseDevice>().ToConstant(MouseDevice)
                 .Bind<IPlatformSettings>().ToSingleton<PlatformSettings>()
                 .Bind<IRenderLoop>().ToConstant(new RenderLoop())
                 .Bind<PlatformHotkeyConfiguration>().ToSingleton<PlatformHotkeyConfiguration>();
-
         }
 
        
         internal static LinuxFramebufferLifetime Initialize<T>(T builder, IOutputBackend outputBackend) where T : AppBuilderBase<T>, new()
         {
-            var platform = new LinuxFramebufferPlatform(outputBackend);
-            builder.UseSkia().UseWindowingSubsystem(platform.Initialize, "fbdev");
-            return new LinuxFramebufferLifetime(platform._fb);
+            var platform = new LinuxFramebufferPlatform();
+            
+            builder
+                .UseSkia()
+                .UseWindowingSubsystem(platform.Initialize, OutputPlatformName(outputBackend));
+            
+            return new LinuxFramebufferLifetime(outputBackend);
+        }
+
+        private static string OutputPlatformName(IOutputBackend outputBackend)
+        {
+            var name = outputBackend.GetType().Name.ToLowerInvariant();
```

Comment:
That should be a property on IOutputBackend

Reply:
Done